pkg/kubelet/kubelet.go

```diff
@@ -1863,11 +1863,11 @@ func (kl *Kubelet) SyncPod(ctx context.Context, updateType kubetypes.SyncPodType
 		if !pcm.Exists(pod) && !firstSync {
 			p := kubecontainer.ConvertPodStatusToRunningPod(kl.getRuntime().Type(), podStatus)
 			if err := kl.killPod(ctx, pod, p, nil); err == nil {
-				if wait.Interrupted(err) {
-					return false, err
-				}
 				podKilled = true
 			} else {
+				if wait.Interrupted(err) {
```

Comment:
```suggestion
				if wait.Interrupted(err) {
				        klog.ErrorS(err, "KillPod interrupted", "pod", klog.KObj(pod), "podStatus", podStatus)
```

Reply:
>Any early interruption should result in exiting the Sync*Pod method as quickly as possible without logging intermediate errors.

should we maintain consistency with other places in Sync*Pod where `wait.Interrupted` is called?

Reply:
agreed.

Reply:
/lgtm